docs/staking-modules/csm/rewards.md

```diff
@@ -41,6 +41,23 @@ The `frame` for the Performance Oracle report is set to 28 days. This makes the
 
 The performance threshold is relative to the overall network attestation effectiveness to ensure that network issues outside the Node Operator's control do not affect reward allocation.
 
+Performance Oracle creates a few artifacts for each successful round of reward distribution: a dump of a Merkle Tree with Node Operators' cumulative rewards and a log of per-operator performance assessment data.
+
+Both files are uploaded to IPFS, and their corresponding CIDs (essentially hashes of the files used to retrieve the content back from the IPFS network) are pushed on-chain. The [`CSFeeDistributor` contract](/staking-modules/csm/contracts/CSFeeDistributor.md) has two view functions to retrieve these CIDs: [**treeCid**](/staking-modules/csm/contracts/CSFeeDistributor.md#treecid) and [**logCid**](/staking-modules/csm/contracts/CSFeeDistributor.md#logcid).
+
+The Merkle tree dump can be used to construct a valid proof for Node Operators to claim their acquired rewards. For pre-generated proofs, see the [csm-rewards](https://github.com/lidofinance/csm-rewards) GitHub repository. This repository also provides detailed instructions on how to generate proof and claim rewards manually via Etherscan.
+
+A frame performance assessment log aims to achieve more transparency on rewards distribution made by Oracles. It's another JSON object that stores, among other things:
+
+- The performance threshold for a given frame;
+- The total amount of shares distributable in the frame;
+- Attestation rates of validators as "assigned" and "included" pairs;
+- The amount of shares distributed every operator in the frame.
+
+There's also additional data in the log; for a full definition look at the following [typescript gist](https://gist.github.com/madlabman/33bc63843b633aa114173d7898e5fcce).
```

Comment:
not sure if it's a good idea to refer to the personal gist :new_moon_with_face:

Reply:
The gist has been moved to CSM repo